### DISCUSSION TITLE
v3.3.0 Beta - Custom Theming overhaul

### DISCUSSION THREAD

**Discussion by SethBurkart123** [SethBurkart123/EvenBetterSEQTA#138] [Announcements]
## What's Changed
* Introduced overhauled theming system which allows you to create custom themes and also a fully fledged store for downloading others (we need to make more themes, I've made one for an example in the meantime).

<hr /><em>This discussion was created from the release <a href='https://github.com/BetterSEQTA/BetterSEQTA-Plus/releases/tag/v3.3.0-beta'>v3.3.0 Beta - Custom Theming overhaul</a>.</em>

**Comment by MEGA-Dawg68** [comment]
10/10 for the theme store! fits in so well!

**Comment by 97-42** [comment]
We could probably give users the ability to publish their own themes. I just think it would be interesting to see what the community could make.

**Reply by SethBurkart123** [reply]
Yes, that's definitely the idea. But as the store is basically run using github as our backend it's a little complicated. We need to write up some documentation showing people how to submit, but the plan was to do it via discord (or I guess you could do it directly on github if you wanted).

**Reply by ar-cyber** [reply]
> Yes, that's definitely the idea. But as the store is basically run using github as our backend it's a little complicated. We need to write up some documentation showing people how to submit, but the plan was to do it via discord (or I guess you could do it directly on github if you wanted).

Like a Discord bot to do so?

**Reply by ar-cyber** [reply]
> Yes, that's definitely the idea. But as the store is basically run using github as our backend it's a little complicated. We need to write up some documentation showing people how to submit, but the plan was to do it via discord (or I guess you could do it directly on github if you wanted).

I'd guess that <13 people use this extension, so there wouldn't be any point to publish using Discord. It may be a good idea to do both.
